src/pages/Items.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from "react";
+import heartIcon from "../assets/ic_heart.png";
+import "../styles/items.css";
+
+import ItemTop from "../components/ItemTop";
+import Pagnation from "../components/Pagination";
+import useMediaQuery from "../utils/useQueryMedia";
+
+const filterName = {
+  recent: "최신순",
+  favorite: "좋아요순",
+};
```

Comment:
값이 고정된 데이터라면 
```suggestion
const FILTER_NAME = {
  recent: "최신순",
  favorite: "좋아요순",
};
```

UPPER_SNAKE_CASE로 이름짓는게 일반적입니다.
변하지않는 상수를 뜻해요.